### PR TITLE
Update test files to use PHPUnit6 notiation for classes

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
These are changes made after doing local runs with phpunit6 there is still an outstanding issue of the YAML Symphony component not being found when running Cccase test but not sure why that is but this is to facilitate testing at least

Before
----------------------------------------
_The current status. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._

After
----------------------------------------
_What has been changed. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._

Technical Details
----------------------------------------
_If the PR introduces noteworthy technical changes, please describe them here. Provide code snippets if necessary_

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

**Comment by seamuslee001** [comment]
ping @eileenmcnaughton

File: tests/phpunit/CRM/Core/ComposerConfigTest.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Class CRM_Core_ComposerConfigTest
  * @group headless
  */
-class CRM_Core_ComposerConfigTest extends \PHPUnit_Framework_TestCase {
+class CRM_Core_ComposerConfigTest extends \PHPUnit\Framework\TestCase {
```
Comment (eileenmcnaughton) [inline tests/phpunit/CRM/Core/ComposerConfigTest.php]: So specifically these are safe
https://engineering.facile.it/blog/eng/phpunit-upgrade-namespace/

**Comment by eileenmcnaughton** [comment]
@seamuslee001 style warning

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by eileenmcnaughton** [comment]
```
ok 127 - Civi\Core\SettingsStackTest::testStack
not ok 128 - Failure: Civi\Test\ExampleHookTest::testPageOutput
  ---
  message: 'Failed asserting that '''
  severity: fail
  ...
ok 129 - Civi\Test\ExampleTransactionalTest::testDummy1
not ok 130 - Failure: Civi\Test\ExampleTransactionalTest::testDummy2
  ---
  message: 'Failed asserting that true is false.'
  severity: fail
  ...

Fatal error: Uncaught exception 'Exception' with message 'Uh oh! testDummy1 still exists!' in /home/jenkins/bknix-dfl/build/core-14387-2hbjk/sites/all/modules/civicrm/tests/phpunit/Civi/Test/ExampleTransactionalTest.php:84
Stack trace:
```

**Comment by seamuslee001** [comment]
@eileenmcnaughton i think i have them fixed now

**Comment by eileenmcnaughton** [comment]
I'll believe it when jenkins says yes!

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
@eileenmcnaughton Jenkins seems happy now

**Comment by eileenmcnaughton** [comment]
jenkins knows best - we need to tell people that actually it's php unit 5.7 min now & I jumped the gun?

**Pull request merged**